src/LightSpeed/toolbox.js

```diff
@@ -83,8 +83,8 @@ var LSToolbox = GObject.registerClass(class LSToolbox extends Toolbox {
     }
 
     _onReset() {
-        Gio.DBus.session.call_sync('com.endlessm.LightSpeed',
-            '/com/endlessm/LightSpeed', 'org.gtk.Actions', 'Activate',
+        Gio.DBus.session.call_sync('com.hack_computer.LightSpeed',
```

Comment:
There are instances of `com.endlessm.LightSpeed` in the clubhouse. Try doing `git grep "com.endlessm.LightSpeed"` in the clubhouse repository.